test/pet.test.js

```diff
@@ -4,7 +4,7 @@ const Pet = require('../src/pet');
   let petCat;
 
   beforeEach(() => {
```

Comment:
Before I proceed with the review of your tests, let's rewrite a test and see what happens. In your `describe('growUp'`) - set `petCat = 31;` and then call the `growUp()`. Then tell me if the age is increased by 31 or if it flags the error. We can go from there!